--- minicom-2.2/src/minicom.c.orig	2005-10-31 13:13:51.000000000 +0100
+++ minicom-2.2/src/minicom.c	2006-10-25 19:56:27.952688500 +0200
@@ -1124,6 +1124,8 @@
           use_status = 1;
           break;
         case 'C': /* Capturing */
+	  setegid(real_gid);
+	  seteuid(real_uid);
           capfp = sfopen(optarg, "a");
           if (capfp == NULL) {
             werror(_("Cannot open capture file"));
@@ -1131,6 +1133,8 @@
           }
           docap = 1;
           vt_set(addlf, -1, docap, -1, -1, -1, -1);
+	  seteuid(eff_uid);
+	  setegid(eff_gid);
           break;
         case 'S': /* start Script */
           strncpy(scr_name, optarg, 33);
