--- minicom-2.8/src/updown.c.orig	2021-01-01 18:45:55.000000000 +0100
+++ minicom-2.8/src/updown.c	2022-01-27 20:13:24.765495507 +0100
@@ -406,6 +406,8 @@ void updown(int what, int nr)
   }
 
   while (udpid != m_wait(&status));
+  if (status)
+    werror(_("Error executing `%s'"), cmdline);
   if (win) {
     enab_sig(0, 0);
     signal(SIGINT, SIG_IGN);
@@ -546,6 +548,8 @@ void kermit(void)
   }
 
   m_wait(&status);
+  if (status)
+    werror(_("Error executing `%s'"), P_KERMIT);
 
   /* Restore screen and keyboard modes */
   mc_wreturn();
@@ -784,6 +788,8 @@ void runscript(int ask, const char *s, c
 
   /* Collect status, and clean up. */
   m_wait(&status);
+  if (status)
+    werror(_("Error executing `%s'"), cmdline);
   enab_sig(0, 0);
   signal(SIGINT, SIG_IGN);
   setcbreak(2); /* Raw, no echo */
