diff -Naur minicom-1.83.1/src/minicom.c minicom-1.83.1-p/src/minicom.c
--- minicom-1.83.1/src/minicom.c	Sat May  5 08:47:36 2001
+++ minicom-1.83.1-p/src/minicom.c	Sat May  5 08:49:05 2001
@@ -1449,7 +1449,7 @@
 	m_restorestate(portfd);
   else {
         if (P_LOGCONN[0] == 'Y' && online > 0)
-	  do_log(_("Quit without reset while online."));
+	  do_log("%s", _("Quit without reset while online."));
 	m_hupcl(portfd, 0);
   }
   if (capfp != (FILE *)0) fclose(capfp);
diff -Naur minicom-1.83.1/src/updown.c minicom-1.83.1-p/src/updown.c
--- minicom-1.83.1/src/updown.c	Thu Mar 16 10:44:26 2000
+++ minicom-1.83.1-p/src/updown.c	Sat May  5 08:50:30 2001
@@ -235,7 +235,7 @@
 #endif
 
   if (P_LOGXFER[0] == 'Y')
-    do_log(cmdline);   /* jl 22.06.97 */
+    do_log("%s", cmdline);   /* jl 22.06.97 */
 
   if (P_PFULL(g) == 'N') {
     win = wopen(10, 7, 70, 13, BSINGLE, stdattr, mfcolor, mbcolor, 1, 0, 1);
@@ -302,11 +302,11 @@
 			!strncmp (buffirst, "Sending", 7)) {
 		      if (xfrstr[0]) {
 			trim (trimbuf, xfrstr, sizeof(trimbuf));
-			do_log (trimbuf);
+			do_log ("%s", trimbuf);
 			xfrstr[0] = 0;
 		      }
 		      trim (trimbuf, buf, sizeof(trimbuf));
-		      do_log(trimbuf);
+		      do_log("%s", trimbuf);
 		    } else if (!strncmp (buffirst, "Bytes", 5)) {
 		      strncpy (xfrstr, buf, sizeof(xfrstr));
 		    }
@@ -323,7 +323,7 @@
   /* Log the last file size	jl 14.09.97 */
   if (P_LOGXFER[0] == 'Y' && xfrstr[0]) {
     trim (trimbuf, xfrstr, sizeof(trimbuf));
-    do_log (trimbuf);
+    do_log ("%s", trimbuf);
     xfrstr[0] = 0;
   }
 
