diff -urN minicom-1.82.orig/src/config.c minicom-1.82/src/config.c
--- minicom-1.82.orig/src/config.c	Thu Dec 24 23:02:25 1998
+++ minicom-1.82/src/config.c	Thu Dec 24 23:06:20 1998
@@ -536,7 +536,7 @@
 {
   WIN *w;
   char *str;
-  int c, x, y, ypos, maxl, string_size;
+  static int c, x, y, ypos, maxl, string_size;
   char* init_string = _(" A - Init string ........."),
       * reset_string = _(" B - Reset string ........"),
       * dialing_prefix_1 = _(" C - Dialing prefix #1...."),
@@ -1258,7 +1258,7 @@
 char *pa;
 int curr_ok;
 {
-  int c;
+  static int c;
   WIN *w;
   int x, y;
   int max = m_getmaxspd();
diff -urN minicom-1.82.orig/src/minicom.c minicom-1.82/src/minicom.c
--- minicom-1.82.orig/src/minicom.c	Thu Dec 24 23:02:26 1998
+++ minicom-1.82/src/minicom.c	Thu Dec 24 23:02:49 1998
@@ -433,7 +433,7 @@
   char tmp_line[MAXCOLS];
 #endif /*SEARCH_HISTORY*/
   int citemode = 0;
-  int cite_ystart = 1000000,
+  static int cite_ystart = 1000000,
       cite_yend = -1,
       cite_y;
   int inverse;
