diff -Nru minicom-1.83.1/src/config.h minicom-1.83.1.new/src/config.h
--- minicom-1.83.1/src/config.h	Tue May 30 09:24:32 2000
+++ minicom-1.83.1.new/src/config.h	Tue May 30 09:26:12 2000
@@ -19,7 +19,7 @@
  */
 
 /* Operating system INdependant parameters. (Usually the same everywhere) */
-#define KERMIT "/usr/local/bin/kermit -l %l -b %b"	/* How to call kermit */
+#define KERMIT "/usr/bin/kermit -l %l -b %b"	/* How to call kermit */
 #ifdef SVR4_LOCKS
 #	define UUCPLOCK	"/var/spool/locks"	/* SVR4 version of Lock file directory */
 #else
@@ -29,8 +29,8 @@
 
 /* Operating system dependant parameters, per OS. A few samples are given. */
 #if defined(__linux__)
-#  define DFL_PORT "/dev/ttyS1"		/* Which tty to use */
-#  define DEF_BAUD "57600"		/* Default baud rate */
+#  define DFL_PORT "/dev/modem"		/* Which tty to use */
+#  define DEF_BAUD "38400"		/* Default baud rate */
 #  define CALLOUT  ""			/* Gets run to get into dial out mode */
 #  define CALLIN   ""			/* Gets run to get into dial in mode */
 #  undef  UUCPLOCK
diff -Nru minicom-1.83.1/src/install.sh minicom-1.83.1.new/src/install.sh
--- minicom-1.83.1/src/install.sh	Tue May 30 09:24:32 2000
+++ minicom-1.83.1.new/src/install.sh	Tue May 30 09:23:12 2000
@@ -3,11 +3,6 @@
 #
 NAME=`id |sed 's/uid=[0-9]*(\(.*\))  *gid=.*$/\1/'`
 
-if test "$NAME" != root
-then
-	echo "You must be root to install minicom."
-	exit 1
-fi
 
 if test $# != 4
 then
@@ -46,8 +41,6 @@
 then
 	echo "Installing minicom in $2"
 	cp minicom $2/minicom
-	chown root $2/minicom
-	chgrp uucp $2/minicom
 	chmod 755 $2/minicom
 fi
 
@@ -58,8 +51,6 @@
 	echo "Installing $i in $2"
 	cp $i $2/$i
 	chmod 755 $2/$i
-	chown root $2/$i
-	chgrp root $2/$i
   fi
 done
 
@@ -68,8 +59,6 @@
 	echo "Installing keyserv in $1"
 	cp keyserv $1/keyserv
 	chmod 755 $1/keyserv
-	chown root $1/keyserv
-	chgrp root $1/keyserv
 fi
 
 echo "Installing manpages in $3"
@@ -77,16 +66,12 @@
 do
 	cp ../man/$i $3
 	chmod 644 $3/$i
-	chown root $3/$i
-	chgrp root $3/$i
 done
 
 if [ ! -f $1/minicom.users ]
 then
 	echo "Installing sample config file minicom.users in $1"
 	cp minicom.users $1
-	chown root $1/minicom.users
-	chgrp root $1/minicom.users
 	chmod 644 $1/minicom.users
 fi
 
