--- minicom-2.2/configure.in.orig	2006-10-25 20:16:10.658603000 +0200
+++ minicom-2.2/configure.in	2006-10-25 20:22:45.891303500 +0200
@@ -157,7 +157,7 @@
 
 dnl Checks for header files.
 dnl Do not use <ncurses/termcap.h> unless we're linking with ncurses.
-if test "x$termlib" = xncurses; then
+if test "x$termlib" = xncurses -o "x$termlib" = xtinfo; then
   dnl Use AC_CHECK_HEADERS so the HAVE_*_H symbol gets defined.
   AC_CHECK_HEADERS(ncurses/termcap.h)
 fi
--- minicom-2.3/src/window.c.orig	2008-09-05 12:44:15.000000000 +0200
+++ minicom-2.3/src/window.c	2008-09-05 15:03:07.000000000 +0200
@@ -26,6 +26,13 @@
 #include <stdarg.h>
 #include <wchar.h>
 
+#ifdef HAVE_TERMCAP_H
+#  include <termcap.h>
+#endif
+#ifdef HAVE_NCURSES_TERMCAP_H
+#  include <ncurses/termcap.h>
+#endif
+
 #include "port.h"
 #include "minicom.h"
 #include "intl.h"

