--- minicom-2.2/src/updown.c.orig	2005-08-14 22:39:30.000000000 +0200
+++ minicom-2.2/src/updown.c	2006-10-25 20:03:31.063131250 +0200
@@ -332,6 +332,8 @@
   }
 
   while (udpid != m_wait(&status));
+  if (status)
+    werror(_("Error executing `%s'"), cmdline);
   if (win) {
     enab_sig(0, 0);
     signal(SIGINT, SIG_IGN);
@@ -409,6 +411,8 @@
   }
 
   m_wait(&status);
+  if (status)
+    werror(_("Error executing `%s'"), P_KERMIT);
 
   /* Restore screen and keyboard modes */
   wreturn();
@@ -638,6 +642,8 @@
   
   /* Collect status, and clean up. */
   m_wait(&status);
+  if (status)
+    werror(_("Error executing `%s'"), cmdline);
   enab_sig(0, 0);
   signal(SIGINT, SIG_IGN);
   setcbreak(2); /* Raw, no echo */

