diff -ur minicom-2.00.0.orig/src/config.c minicom-2.00.0/src/config.c
--- minicom-2.00.0.orig/src/config.c	Sat Oct 28 14:51:43 2000
+++ minicom-2.00.0/src/config.c	Thu May 16 21:20:55 2002
@@ -619,7 +619,7 @@
 {
   WIN *w;
   char *str;
-  int c, x, y, ypos, maxl, string_size;
+  static int c, x, y, ypos, maxl, string_size;
   char* init_string = _(" A - Init string ........."),
       * reset_string = _(" B - Reset string ........"),
       * dialing_prefix_1 = _(" C - Dialing prefix #1...."),
@@ -1372,7 +1372,7 @@
 char *stopb;
 int curr_ok;
 {
-  int c;
+  static int c;
   WIN *w;
   int x, y;
   int max = m_getmaxspd();
diff -ur minicom-2.00.0.orig/src/minicom.c minicom-2.00.0/src/minicom.c
--- minicom-2.00.0.orig/src/minicom.c	Wed Jul  4 18:27:47 2001
+++ minicom-2.00.0/src/minicom.c	Thu May 16 21:20:55 2002
@@ -441,7 +441,7 @@
   char tmp_line[MAXCOLS];
 #endif /*SEARCH_HISTORY*/
   int citemode = 0;
-  int cite_ystart = 1000000,
+  static int cite_ystart = 1000000,
       cite_yend = -1,
       cite_y;
   int inverse;
