diff -Naur minicom-1.83.0/src/rwconf.c minicom-1.83.0-new/src/rwconf.c
--- minicom-1.83.0/src/rwconf.c	Tue Apr 20 20:50:19 1999
+++ minicom-1.83.0-new/src/rwconf.c	Fri Feb 25 22:07:42 2000
@@ -73,12 +73,12 @@
   { "",			PUBLIC,   "pname11" },
   { "",			PUBLIC,   "pname12" },
 #ifdef __linux__
-  { "/usr/bin/sz -vv -b",	PUBLIC,   "pprog1" },
-  { "/usr/bin/sb -vv",		PUBLIC,   "pprog2" },
-  { "/usr/bin/sx -vv",		PUBLIC,   "pprog3" },
-  { "/usr/bin/rz -vv -b -E",	PUBLIC,   "pprog4" },
-  { "/usr/bin/rb -vv",		PUBLIC,   "pprog5" },
-  { "/usr/bin/rx -vv",		PUBLIC,   "pprog6" },
+  { "/usr/bin/lsz -vv -b",	PUBLIC,   "pprog1" },
+  { "/usr/bin/lsb -vv",		PUBLIC,   "pprog2" },
+  { "/usr/bin/lsx -vv",		PUBLIC,   "pprog3" },
+  { "/usr/bin/lrz -vv -b -E",	PUBLIC,   "pprog4" },
+  { "/usr/bin/lrb -vv",		PUBLIC,   "pprog5" },
+  { "/usr/bin/lrx -vv",		PUBLIC,   "pprog6" },
   { "/usr/bin/kermit -i -l %l -s", PUBLIC, "pprog7" },
   { "/usr/bin/kermit -i -l %l -r", PUBLIC, "pprog8" },
 #else
