diff -ru minicom-1.82.1/src/Makefile minicom-1.82.1.AX/src/Makefile
--- minicom-1.82.1/src/Makefile	Sat Jul 25 21:05:46 1998
+++ minicom-1.82.1.AX/src/Makefile	Mon Jun 28 22:45:31 1999
@@ -60,7 +60,7 @@
 # MANDIR	= /usr/local/man/man1
 
 # Take these compilation flags for Linux with libncurses.
-FLAGS	= -Wall -D_POSIX -D_SYSV -D_SELECT -pipe # -I/usr/include/ncurses
+FLAGS	= -Wall -D_POSIX -D_SYSV -D_SELECT -pipe -I/usr/include/ncurses
 PROGS	= minicom runscript ascii-xfr
 LFLAGS	= -s
 LIBS	= -lncurses #-lintl
